03/21/2020 11:51:50 [Information]  App starting up
03/21/2020 11:51:57 [Warning]  User "619e8ecd-e2f6-4949-9de0-a4603328da29" is already in role "Admin".
03/21/2020 11:51:57 [Warning]  User "219bfb58-be00-498a-9b11-aace35148830" is already in role "User".
03/21/2020 11:51:57 [Warning]  User "80e369c6-63ce-453d-a537-7e21966dde31" validation failed: "InvalidUserName".
03/21/2020 11:51:57 [Warning]  User "80e369c6-63ce-453d-a537-7e21966dde31" validation failed: "InvalidUserName".
03/21/2020 11:52:05 [Error]  Browserslist: caniuse-lite is outdated. Please run next command `npm update`

03/21/2020 11:52:29 [Information]  HTTP "GET" "/" responded 200 in 33661.2746 ms
03/21/2020 11:52:31 [Information]  HTTP "GET" "/runtime.js" responded 200 in 2067.2050 ms
03/21/2020 11:52:31 [Information]  HTTP "GET" "/main.js" responded 200 in 2069.6039 ms
03/21/2020 11:52:31 [Information]  HTTP "GET" "/polyfills.js" responded 200 in 2082.2176 ms
03/21/2020 11:52:31 [Information]  HTTP "GET" "/styles.css" responded 200 in 2082.2066 ms
03/21/2020 11:52:31 [Information]  HTTP "GET" "/vendor.js" responded 200 in 2140.8913 ms
03/21/2020 11:52:33 [Information]  HTTP "GET" "/runtime.js.map" responded 200 in 2022.2407 ms
03/21/2020 11:52:33 [Information]  HTTP "GET" "/polyfills.js.map" responded 200 in 2023.1128 ms
03/21/2020 11:52:33 [Information] TestLibraryNewVersion.Controllers.BooksController.GetBooks (TestLibraryNewVersion) Get Books
03/21/2020 11:52:33 [Error] TestLibraryNewVersion.Controllers.BooksController.GetBooks (TestLibraryNewVersion) Error Get Books
03/21/2020 11:52:34 [Information] TestLibraryNewVersion.Controllers.BooksController.GetBooks (TestLibraryNewVersion) HTTP "GET" "/api/Books" responded 200 in 420.3285 ms
03/21/2020 11:52:35 [Information]  HTTP "GET" "/main.js.map" responded 200 in 2020.9542 ms
03/21/2020 11:52:35 [Information]  HTTP "GET" "/vendor.js.map" responded 200 in 2064.6562 ms
03/21/2020 11:52:35 [Information]  HTTP "GET" "/sockjs-node/info?t=1584784353839" responded 200 in 2031.0542 ms
03/21/2020 11:53:20 [Error] TestLibraryNewVersion.Controllers.AuthController.Login (TestLibraryNewVersion) HTTP "POST" "/api/Auth/login" responded 500 in 610.1038 ms
System.ArgumentNullException: Value cannot be null. (Parameter 'userName')
   at Microsoft.AspNetCore.Identity.UserManager`1.FindByNameAsync(String userName)
   at Library.BLL.Services.AuthenticationService.FindByName(String name) in C:\Users\user\Documents\Visual Studio 2019\Projects\TestLibraryNewVersion\Library.BLL\Services\AuthenticationService.cs:line 18
   at TestLibraryNewVersion.Controllers.AuthController.Login(UserLoginDTO userLogin) in C:\Users\user\Documents\Visual Studio 2019\Projects\TestLibraryNewVersion\TestLibraryNewVersion\Controllers\AuthController.cs:line 39
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
03/21/2020 11:53:20 [Error]  An unhandled exception has occurred while executing the request.
System.ArgumentNullException: Value cannot be null. (Parameter 'userName')
   at Microsoft.AspNetCore.Identity.UserManager`1.FindByNameAsync(String userName)
   at Library.BLL.Services.AuthenticationService.FindByName(String name) in C:\Users\user\Documents\Visual Studio 2019\Projects\TestLibraryNewVersion\Library.BLL\Services\AuthenticationService.cs:line 18
   at TestLibraryNewVersion.Controllers.AuthController.Login(UserLoginDTO userLogin) in C:\Users\user\Documents\Visual Studio 2019\Projects\TestLibraryNewVersion\TestLibraryNewVersion\Controllers\AuthController.cs:line 39
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
03/21/2020 11:53:33 [Error] TestLibraryNewVersion.Controllers.AuthController.Login (TestLibraryNewVersion) HTTP "POST" "/api/Auth/login" responded 500 in 64.8672 ms
System.ArgumentNullException: Value cannot be null. (Parameter 'userName')
   at Microsoft.AspNetCore.Identity.UserManager`1.FindByNameAsync(String userName)
   at Library.BLL.Services.AuthenticationService.FindByName(String name) in C:\Users\user\Documents\Visual Studio 2019\Projects\TestLibraryNewVersion\Library.BLL\Services\AuthenticationService.cs:line 18
   at TestLibraryNewVersion.Controllers.AuthController.Login(UserLoginDTO userLogin) in C:\Users\user\Documents\Visual Studio 2019\Projects\TestLibraryNewVersion\TestLibraryNewVersion\Controllers\AuthController.cs:line 39
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
03/21/2020 11:53:33 [Error]  An unhandled exception has occurred while executing the request.
System.ArgumentNullException: Value cannot be null. (Parameter 'userName')
   at Microsoft.AspNetCore.Identity.UserManager`1.FindByNameAsync(String userName)
   at Library.BLL.Services.AuthenticationService.FindByName(String name) in C:\Users\user\Documents\Visual Studio 2019\Projects\TestLibraryNewVersion\Library.BLL\Services\AuthenticationService.cs:line 18
   at TestLibraryNewVersion.Controllers.AuthController.Login(UserLoginDTO userLogin) in C:\Users\user\Documents\Visual Studio 2019\Projects\TestLibraryNewVersion\TestLibraryNewVersion\Controllers\AuthController.cs:line 39
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
03/21/2020 11:56:00 [Error] TestLibraryNewVersion.Controllers.AuthController.Login (TestLibraryNewVersion) HTTP "POST" "/api/Auth/login" responded 500 in 148.9380 ms
System.ArgumentNullException: Value cannot be null. (Parameter 'userName')
   at Microsoft.AspNetCore.Identity.UserManager`1.FindByNameAsync(String userName)
   at Library.BLL.Services.AuthenticationService.FindByName(String name) in C:\Users\user\Documents\Visual Studio 2019\Projects\TestLibraryNewVersion\Library.BLL\Services\AuthenticationService.cs:line 18
   at TestLibraryNewVersion.Controllers.AuthController.Login(UserLoginDTO userLogin) in C:\Users\user\Documents\Visual Studio 2019\Projects\TestLibraryNewVersion\TestLibraryNewVersion\Controllers\AuthController.cs:line 39
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
03/21/2020 11:56:00 [Error]  An unhandled exception has occurred while executing the request.
System.ArgumentNullException: Value cannot be null. (Parameter 'userName')
   at Microsoft.AspNetCore.Identity.UserManager`1.FindByNameAsync(String userName)
   at Library.BLL.Services.AuthenticationService.FindByName(String name) in C:\Users\user\Documents\Visual Studio 2019\Projects\TestLibraryNewVersion\Library.BLL\Services\AuthenticationService.cs:line 18
   at TestLibraryNewVersion.Controllers.AuthController.Login(UserLoginDTO userLogin) in C:\Users\user\Documents\Visual Studio 2019\Projects\TestLibraryNewVersion\TestLibraryNewVersion\Controllers\AuthController.cs:line 39
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
03/21/2020 11:56:16 [Information]  HTTP "GET" "/polyfills.js.map" responded 304 in 2056.9756 ms
03/21/2020 11:56:16 [Information]  HTTP "GET" "/runtime.js.map" responded 304 in 2068.8698 ms
03/21/2020 11:56:16 [Information]  HTTP "GET" "/vendor.js.map" responded 304 in 2069.8455 ms
03/21/2020 11:56:16 [Information]  HTTP "GET" "/main.js.map" responded 200 in 2064.5693 ms
03/21/2020 11:56:16 [Information]  HTTP "GET" "/styles.css" responded 304 in 2067.0862 ms
03/21/2020 11:56:16 [Information] TestLibraryNewVersion.Controllers.AuthController.Login (TestLibraryNewVersion) HTTP "POST" "/api/Auth/login" responded 200 in 562.7769 ms
03/21/2020 11:56:16 [Information] TestLibraryNewVersion.Controllers.BooksController.GetBooks (TestLibraryNewVersion) Get Books
03/21/2020 11:56:16 [Error] TestLibraryNewVersion.Controllers.BooksController.GetBooks (TestLibraryNewVersion) Error Get Books
03/21/2020 11:56:17 [Information] TestLibraryNewVersion.Controllers.BooksController.GetBooks (TestLibraryNewVersion) HTTP "GET" "/api/Books" responded 200 in 14.9478 ms
03/21/2020 11:56:18 [Information]  HTTP "GET" "/styles.css.map" responded 304 in 2019.9976 ms
03/21/2020 11:57:46 [Information] TestLibraryNewVersion.Controllers.AuthController.Login (TestLibraryNewVersion) HTTP "POST" "/api/Auth/login" responded 200 in 28.5870 ms
